src/header/HeaderSystemMenu.js

```diff
@@ -36,8 +36,10 @@ import './HeaderSystemMenu.less';
 /**
  * HeaderSystemMenu component is a button when clicked this button reveals the
  * links to System menu options. <br>
- * With user access permission, Download logs is an available option to retrieve
- * all logs as a single file. <br>
+ * With user access permission, two options for retrieving logs are available.
+ * "Download logs" retrieves all logs as a single file, and "View logs" allows
+ * the 1000 lines of the log to be viewed in the UI, where they can be
```

Comment:
How about
`View logs shows the last 1000 lines of relevant Gateway logs, providing searching and filtering options. `